### ISSUE TITLE
Missing Shadow Definitions for `cython.operator`

### ISSUE THREAD

**Issue by rickmark** [cython/cython#5277]
### Describe the bug

There are no exported shadows for `cython.operator` leading any IDE or linter / type checker to believe those functions such as dereference / preincrement etc do not exist.

### Code to reproduce the behaviour:

```cython
# example code

from cython.operator cimport dereference
```

### Expected behaviour

IDE should be aware of dereference/preincrement etc from typing / shadow

### Environment

OS: [e.g. Linux, Windows, macOS]
Python version [e.g. 3.10.2]
Cython version [e.g. 3.0.0a11]

**Comment by 0dminnimda** [comment]
What IDE/tool are you using to parse and typecheck cython code?

**Comment by da-woods** [comment]
Yes - these would definitely be good to have. It's unfortunately probably difficult to make them behave sensibly though
